test/src/unit-serialization.cpp

```diff
@@ -189,3 +189,20 @@ TEST_CASE("serialization")
         test("[3,\"false\",false]", "[3,\\\"false\\\",false]");
     }
 }
+
+TEST_CASE_TEMPLATE("serialization for extreme integer values", T, int32_t, uint32_t, int64_t, uint64_t)
+{
+    SECTION("minimum")
+    {
+        static constexpr auto minimum = (std::numeric_limits<T>::min)();
```

Comment:
Out of curiosity: why is this variable static?

Reply:
That is a left over from the first version which was only `static const`. Thanks for spotting. From my understanding there should be no benefit from `static constexpr` compared to `constexpr`.